pycalphad/core/custom_autowrap.py

```diff
@@ -102,7 +101,7 @@ def import_extension(path, modname):
     # Blacklist fixes gh-65.
     # We filter out any files that can be created by compilers which are not our actual compiled file.
     # We cannot more directly search for our files because of differing platforms.
-    blacklist = ['dSYM']
+    blacklist = ['dSYM', 'c', 'pyx']
```

Comment:
Yes. There may be a way to control it as a command-line option, but I liked having everything in one directory for debugging and it doesn't seem like there's harm in leaving it like this.